packages/mui-system/src/createStyled.js

```diff
@@ -102,6 +117,9 @@ export default function createStyled(input = {}) {
     } else if (componentSlot) {
       // any other slot specified
       shouldForwardPropOption = slotShouldForwardProp;
+    } else if (isStringTag(tag)) {
```

Comment:
We should document that this works only if developers don't use the MUI options (or at least the slot). Otherwise, this would behave differently:

```
styled('a', { slot: 'Root' })``;
styled('a')``
```

Btw, was there a reason for this?

Reply:
Do you mean why it should behave differently?

Reply:
Our use case with slots is more complicated, take a look at Chip as an example:

**Default**
the `<Chip />` root slot is using `div` tag as a default:
```js
```